napari/_qt/containers/_layer_delegate.py

```diff
@@ -56,6 +60,27 @@
     from ...components.layerlist import LayerList
 
 
+@inject_napari_dependencies
+def show_layer_info(ll: LayerList):
+    layer = ll.selection.active
+    dlg = QtLayerInfoDialog(layer=layer)
```

Comment:
```suggestion
def show_layer_info(viewer: Viewer):
    layer = viewer.layers.selection.active
    dlg = QtLayerInfoDialog(layer=layer, parent=viewer.window._qt_window)
```
This is required to dialog inherit style from the napari window.